modules/mod_twitter/mod_twitter.erl

```diff
@@ -92,6 +92,7 @@ is_setting(<<"access_token_secret">>) -> true;
 is_setting(<<"follow">>) -> true;
 is_setting(_) -> false.
 
+-spec observe_rsc_update_done(#rsc_update_done{}, #context{}) -> ok.
```

Comment:
The spec in line 95 uses a record, please define a type for the record and use that instead.